core/frontend/src/tile/map/MapLayerTileTreeReference.ts

```diff
@@ -13,22 +13,42 @@ import { IModelApp } from "../../IModelApp";
 import { IModelConnection } from "../../IModelConnection";
 import { createModelMapLayerTileTreeReference, MapLayerImageryProvider, TileTreeReference } from "../internal";
 
-/** @internal  */
+/**
+ * A reference to a [[TileTree]] to be used for map layer tiles.
+ * @beta
```

Comment:
All members are exposed here, is it needed?  Many of them not documented,,,

Reply:
I've gone back over these and reverted some members that aren't needed